.github/workflows/build-installer.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: python -m pip install -r ../requirements.txt
       - name: Build exe
         working-directory: ${{ github.workspace }}/installer
-        run: pyinstaller.exe --onefile ./microk8s.spec
+        run: pyinstaller.exe ./microk8s.spec
```

Comment:
I fixed this after I saw [this error in GH actions](https://github.com/canonical/microk8s/runs/6386624112?check_suite_focus=true#step:5:12)